README.md

```diff
@@ -131,6 +131,15 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [PepsiCo](https://www.pepsicojobs.com/internshipsprograms/jobs/241240?lang=en-us&previousLocale=en-US)| Purchase, New York <br> Plano, Texas | Software Development & Engineering Intern (No Sponsorship) |
 | [U.S. Bank](https://careers.usbank.com/global/en/job/2023-0020491/2024-Software-Engineering-Summer-Intern-Early-Careers) | Richfield, MN; and Atlanta, GA | 2024 Software Engineering Summer Intern |
 | Jane Street | New York, NY | [Software Engineer Internship](https://www.janestreet.com/join-jane-street/position/6834728002/) <br/> [Quantitative Trader Internship](https://www.janestreet.com/join-jane-street/position/6850721002/) <br/> [Quantitative Researcher Internship](https://www.janestreet.com/join-jane-street/position/6838982002/) |
+| [SkyWorks](https://careers.skyworksinc.com/job/Austin-Embedded-FW-Summer-Intern-2024-TX-73301/1054957600) | Austin, TX | Embedded FW Summer Intern |
+| [Cole Engineering Services, Inc.](https://workforcenow.adp.com/mascsr/default/mdf/recruitment/recruitment.html?cid=5055dea5-b670-4a19-adb1-2aea04ba5207&ccId=19000101_000001&type=MP&lang=en_US&jobId=472432) | Orlando, FL | Intern (Software) (US Citizenship Required) |
+| [Discover](https://jobs.discover.com/jobs/r292162024-business-technology-internship-program-riverwoods/) | Riverwoods, IL | Business Technology Internship Program (No Sponsorship) |
+| [Tyler Technologies](https://jobs.jobvite.com/tylertech/job/ojyMlfwI) | Moraine, OH | Software Development Intern - Summer/Fall |
+| [Steris](https://careers.steris.com/job/RICHMOND-Software-Engineering-Intern-CA-94804/1054522400/) | Richmond, CA | Software Engineer Intern |
+| [Arkansas Center for Data Sciences](https://jobs.lever.co/acds/c29f04bb-9d15-48df-9ba1-7c0106ac8cda) | Fayetteville, AR | ETL Developer/Junior DBA Intern (No Sponsorship) |
+| [Infoverity](https://infoverity.zohorecruit.com/jobs/Infoverity-USA-Careers/530797000006827069/Summer-Internship-summer-2024-) | Dublin, OH | Summer Internship Program (No Sponsorship) |
+| [Eagle Eye Produce](https://eagleeyeproduce.bamboohr.com/careers/108) | Idaho Falls, ID | Web Developer Intern |
+| [Aquaveo](https://www.aquaveo.com/jobs-geospatial-web-developer-intern) | Provo, UT | Geospatial Web Developer Intern |
```

Comment:
Aquaveo's position doesn't _sound_ like it's a summer internship, in particularly with wording like
> Flexible hours (work hours around class schedule)

and
> 20-30 hours per week

I'll let the other collaborators decide on this one.

Reply:
Agreed that it doesn't _sound_ like one but I can't confirm either. Maybe worth emailing them to ask if we wanna bother with the hassle?

Reply:
Don't think it's worth. We can always just leave it as is and see if someone finds out otherwise.